src/track.py

```diff
@@ -11,17 +11,20 @@
 import motmetrics as mm
 import numpy as np
 import torch
-
 from tracker.multitracker import JDETracker
 from tracking_utils import visualization as vis
 from tracking_utils.log import logger
 from tracking_utils.timer import Timer
 from tracking_utils.evaluation import Evaluator
-import datasets.dataset.jde as datasets
+import lib.datasets.dataset.jde as datasets
 
 from tracking_utils.utils import mkdir_if_missing
 from opts import opts
 
+from multiprocessing import Queue
+from multiprocessing import Value, Array
+from multiprocessing import Process
+from ffmpy import FFmpeg
```

Comment:
I suggest adding the ffmpy library to the file as part of this pull request requirements.txt